homeassistant/components/media_source/local_source.py

```diff
@@ -152,7 +152,7 @@ def _build_item_response(
 
         # Check that it's a media file
         if is_file and (
-            not mime_type or mime_type.split("/")[0] not in MEDIA_MIME_TYPES
+            not mime_type or mime_type.partition("/")[0] not in MEDIA_MIME_TYPES
```

Comment:
Same deal, perhaps the rest of the MIME type needs to `content_type.partition(";")[0]`.